rust/src/lib.rs

```diff
@@ -1774,11 +1774,23 @@ pub enum ScriptHashNamespace {
     PlutusScript,
 }
 
+impl ScriptHashNamespace {
+    pub fn scrip_hash_prefix(&self) -> u8 {
+        match self {
+            ScriptHashNamespace::NativeScript => 0 as u8,
```

Comment:
Do we have potential issue if we define number directly in enum ?
For example: 
pub enum ScriptHashNamespace
{
    NativeScript = 0
}

Reply:
Yup, good point, updated here: https://github.com/Emurgo/cardano-serialization-lib/pull/436/commits/789749c1de0154c51304b64233a7e68d9f1ea957